services/frontend/src/components/settings/settings-page.js

```diff
@@ -96,57 +99,66 @@ export class SettingsPage extends React.Component {
             <Page title="Settings">
                 <div style={{ background: 'white', minHeight: '100%' }}>
                     <Container style={{ paddingTop: '1em', paddingBottom: '1em' }}>
-                        <ListSubheader style={{ paddingLeft: 0, paddingRight: 0 }}>Account</ListSubheader>
+                        <ListHeader title="Account" isBig="True" />
                         <div>
-                            <Typography variant="h5">{_.get(this.props, 'user.name')}</Typography>
-                            <Typography>@{_.get(this.props, 'user.username')}</Typography>
-                            <Typography>{_.get(this.props, 'user.email')}</Typography>
+                            <InputGroup
+                                label="NAME"
+                                content={_.get(this.props, 'user.name')}
```

Comment:
Can you use our custom `Typography` to display these? Name should be big-body and the others are just body1 (which is the default)